### DISCUSSION TITLE
monaco-editor/esm/vs/basic-languages/wgsl/wgsl.js:  e.charCodeAt is not a function

### DISCUSSION THREAD

**Discussion by bhavyashah98** [microsoft/monaco-editor#4207] [Q&A]
### Reproducible in vscode.dev or in VS Code Desktop?

- [ ] Not reproducible in [vscode.dev](https://vscode.dev) or VS Code Desktop

### Reproducible in the monaco editor playground?

- [ ] Not reproducible in [the monaco editor playground](https://microsoft.github.io/monaco-editor/playground.html)

### Reproduction Steps

Monaco-editor  v0.37.1
Next v12.2.1-canary.1

Upgrade monaco version from v0.33.0 to v0.37.1

While running next build getting error 
monaco-editor/esm/vs/basic-languages/wgsl/wgsl.js: e.charCodeAt is not a function

### Actual (Problematic) Behavior

Was working fine with monaco-editor v0.33.0. Breaking for v0.37.1

### Expected Behavior

next build should run fine

**Comment by Yokozuna59** [comment]
Hi @bhavyashah98, I think you need to import the following file instead [src/basic-languages/wgsl/wgsl.contribution.ts](https://github.com/microsoft/monaco-editor/blob/94c055bcbdd49f04a0fa15515e848542a79fb948/src/basic-languages/wgsl/wgsl.contribution.ts):

```typescript
import 'monaco-editor/esm/vs/basic-languages/wgsl/wgsl.contribution.js';
```

It's the file where it would register the language.

**Reply by bhavyashah98** [reply]
Can't make change from internal file imports. Is there any fix version deployed after that for monaco-editor?
